src/Stratis.Bitcoin/P2P/PeerConnectorDiscovery.cs

```diff
@@ -60,8 +60,6 @@ public override void OnStartConnect()
 
         public override async Task OnConnectAsync()
         {
-            this.logger.LogTrace("()");
```

Comment:
Just wondering why remove the trace?

Reply:
I just left the logging in the areas where it became applicable, i.e. in the Ifs further down the code when peer selection fails. Otherwise, when I turned on logging, it was completely clogging up the log file.

Reply:
Yeah but thats the point of having method entry and exit points, its useful for threading issues, but its also up to what makes sense on if we need to put a trace at a given point or not so ok with me.

Reply:
Yeah, if we leave the entry and exit logs here and there aren't any issues with peer selection, we get hundreds of:

```
PeerConnector.OnConnectAsync()
PeerConnector.OnConnectAsync(-)
```

in the logs.